addons/sourcemod/scripting/hosties/lastrequest.sp

```diff
@@ -3652,6 +3662,13 @@ InitializeGame(iPartnersIndex)
 		LogToGame("\"%L\" started a LR game (\"%s\") with \"%L\"", LR_Player_Prisoner, LR_Name, LR_Player_Guard);
 	}
 	
+	//remove armor
+	if (gShadow_LR_RemoveArmor == 1)

```

Comment:
When you CreateConVar'd you did not specify a maximum, so you should only check if the value is positive.  Change it to
if (gShadow_LR_RemoveArmor)
to account for this.